src/controller/SetUpCodePairer.cpp

```diff
@@ -59,6 +60,13 @@ CHIP_ERROR GetPayload(const char * setUpCode, SetupPayload & payload)
 }
 } // namespace
 
+SetUpCodePairer::~SetUpCodePairer()
+{
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+    DeviceLayer::ConnectivityMgr().WiFiPAFCancelConnect();
```

Comment:
This should happen when stopping (PAF) discovery, not in the destructor.

Reply:
https://github.com/project-chip/connectedhomeip/issues/34612